src/components/forms/DebouncedCombobox.tsx

```diff
@@ -30,6 +30,7 @@ interface Props {
   clearButton?: any;
   onClearButtonClick?: any;
   inputLabel?: string;
+  filterRecords?: any;
```

Comment:
This is making following & tracking code down super complex. Please annotate and don't use any.